@anilbey : LGTM. Could you take a quick look at [CI test failures](https://travis-ci.org/BlueBrain/spack/builds/654964549?utm_source=github_status&utm_medium=notification)? We have enabled some more checks and it would be easy to fix/improve package: 

```
>       assert not failing, msg.format(
1832            len(failing), ','.join(failing)
1833        )
1834E       AssertionError: there are 1 packages still using old APIs in this PR [py-bglibpy]
1835E       assert not ['py-bglibpy']
```

--- a/var/spack/repos/builtin/packages/py-bglibpy/package.py
+++ b/var/spack/repos/builtin/packages/py-bglibpy/package.py
@@ -13,7 +13,7 @@ class PyBglibpy(PythonPackage):
     git = "ssh://bbpcode.epfl.ch/sim/BGLibPy"
 
     version('develop', branch='master')
-    
+
     version('4.1.4', commit='e54d294460e5bdf6b9990bc10a4606b412b76d90')
     version('4.2.14', commit='ab36dadee66f7ccd65a0ba564b97dfee91e7668b')
 

--- a/var/spack/repos/builtin/packages/py-bglibpy/package.py
+++ b/var/spack/repos/builtin/packages/py-bglibpy/package.py
@@ -19,7 +19,7 @@ class PyBglibpy(PythonPackage):
 
     depends_on('neuron+python', type='run')
     depends_on('py-h5py~mpi@2.3:', type='run')
-    depends_on('py-bluepy@0.13.2:', type='run')
+    depends_on('py-bluepy', type='run')
     depends_on('py-libsonata', type='run')
 
     def setup_environment(self, spack_env, run_env):

